package.json

```diff
@@ -109,8 +112,9 @@
     "npm-shrinkwrap.json"
   ],
   "scripts": {
-    "test": "NODE_ENV=test jest",
-    "lint": "eslint Examples/ Libraries/",
+    "test": "NODE_ENV=test node_modules/jest-cli/bin/jest.js",
```

Comment:
@cpojer My bad. For some reason it didn't work for me last time I tried. Will update the PR.